### PR TITLE
Support multiple control channels. Fixes #469

### LINKED ISSUE #469: Initial support for multiple control channels in routers

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: events/dispatcher_usage.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func (self *usageEventAdapter) AcceptMetricsMsg(message *metrics_pb.MetricsMessa
 				for usageType, usage := range bucket.Values {
 					evt := &event.UsageEvent{
 						Namespace:        event.UsageEventsNs,
-						Version:          event.UsageEventsVersion,
+						Version:          2,
```
Comment (michaelquigley) [inline events/dispatcher_usage.go:119]: magic number?

File: router/config.go [hunk 2]
```diff
@@ -101,7 +103,7 @@ type Config struct {
 		}
 	}
 	Ctrl struct {
-		Endpoint              *UpdatableAddress
+		Endpoints             []*UpdatableAddress
```
Comment (michaelquigley) [inline router/config.go:106]: definitely would love to see the updated controllers get written to a state file. 👍

**Review by michaelquigley — APPROVED** [review]
👍

File: events/dispatcher_usage.go (hunk 1 above)
Comment (plorenz) [reply events/dispatcher_usage.go:119]: Usually have a constant for this, but we're supporting two versions at the same time, so one gets the constant and the other is hard coded to the old version. At point we'll remove support for the old version.

**Pull request merged**